apps/meeting/src/containers/MeetingProviderWrapper/index.tsx

```diff
@@ -57,8 +59,13 @@ const MeetingProviderWrapper: React.FC = () => {
   };
 
   const getMeetingProviderWrapperWithBGBlur = (children: React.ReactNode) => {
+    let filterCPUUtilization = parseInt(blurOption,10);
+    if (!filterCPUUtilization) {
+      filterCPUUtilization = 40;
+    }
```

Comment:
are we defaulting to 40?

Reply:
yes. I put this as a safety check in case the value was not set properly and the parse returned NaN.